docs/README.md

```diff
@@ -50,11 +50,26 @@ You can adapt the `--build_dir` to set any temporary folder that you prefer. Thi
 the MDX files that will be rendered as the documentation on the main website. You can inspect them in your favorite
 Markdown editor.
 
+## Previewing the documentation
+
+To preview the docs, use the following command:
```

Comment:
I think you need extra packages for this, so maybe also indicate one has to do
```
pip install watchdog
```
before.

Reply:
Ah, I didn't realize `watchdog` needed to be installed separately. I think I either had it locally already or it was installed  by the command from earlier in the README:

```
pip install git+https://github.com/huggingface/doc-builder
```

Perhaps I already had it installed and that's why I didn't see the `ImportError`.

Reply:
Fixed on [cc231de](https://github.com/snehankekre/transformers/commit/cc231de39b0784bbea402699f0d040feb4980759).